synapse/lib/webapp.py

```diff
@@ -62,19 +62,19 @@ def post(self, *args):
 
         boss.initJob( task=(func,args,kwargs), ondone=self._onJobDone )
 
-    def _onJobDone(self, job):
-
-        # a bit of "json friendlyness"
-        ret = {'status':'ok'}
-
+    def format_job_response(self, job):
```

Comment:
rename function to something like: _fmtJobResp for consistency with the rest of the method names